pkgs/development/python-modules/polars/default.nix

```diff
@@ -40,7 +40,10 @@ buildPythonPackage rec {
     #sed -i 's/version = "0.18.0"/version = "${version}"/g' Cargo.lock
   '';
 
-  cargoDeps = rustPlatform.importCargoLock { lockFile = ./Cargo.lock; };
+  cargoDeps = rustPlatform.importCargoLock {
+    lockFile = "${src}/Cargo.lock";
```

Comment:
@happysalada Is there a reason to not just depend directly on the source Cargo.lock here?

Reply:
IFD